src/Autocomplete/src/DependencyInjection/AutocompleteExtension.php

```diff
@@ -149,6 +158,7 @@ private function registerFormServices(ContainerBuilder $container): void
         $container
             ->register('ux.autocomplete.choice_type_extension', AutocompleteChoiceTypeExtension::class)
             ->setArguments([
+                new Reference('ux.autocomplete.checksum_calculator'),
```

Comment:
Maybe second  argument to avoid a BC problem ?

Reply:
In such a case, we will have to make it an optional argument, as translator is nullable and is null by default.

Reply:
Yep my bad, sorry